Tests/SwiftFormatTests/PrettyPrint/Selections.swift

```diff
@@ -0,0 +1,231 @@
+import SwiftFormat
+import XCTest
+
+final class Selections: PrettyPrintTestCase {
+  func testSelectAll() {
+    let input =
+      """
+      ➡️func foo() {
```

Comment:
I wish these arrows showed up better in GitHub 🙁   I see that in Xcode they're rendered as emoji, but GitHub is rendering them using the non-emoji variant, and they're not even the same size in the font that GitHub uses on my machine.

![image](https://github.com/apple/swift-format/assets/1062427/5d5129f9-462a-4e9b-ab14-a7b8e9990e65)

How would you feel about using guillemets (`«...»`) instead? Those are unlikely to appear in Swift code that we'd use in tests, they're a natural bracket form, and they're easy to type on a Mac keyboard in US layout (Option+Backslash and Shift+Option+Backslash).

Reply:
I like guillemets, and it's what I actually thought of first. But I wish they showed up better like the boxed number emojis we're using.

Reply:
FWIW GitHub on Safari shows these arrows as boxes for me and GitHub on Google Chrome on macOS shows the left arrow as a box for me.

---

I would very much like to use some emoji that is rendered using a box because that way they really stand out to humans while looking at the source `«…»` fades too much into the source code for my taste. What do you think about ⏩️ and ⏪️ like we use in swift-syntax here: https://github.com/apple/swift-syntax/blob/55c4e4669c031d697e1924022b8ba250cfde0f2f/Sources/_SwiftSyntaxTestSupport/IncrementalParseTestUtils.swift#L182-L184